x/launch/client/cli/tx_create_chain.go

```diff
@@ -58,6 +62,23 @@ func CmdCreateChain() *cobra.Command {
 				if err != nil {
 					return err
 				}
+				initialGenesis = types.NewGenesisURL(genesisURL, genesisHash)
+			}
+
+			// parse genesis config file for initialGenesis
+			genesisConfigFile, err := cmd.Flags().GetString(flagGenesisConfigFile)
+			if err != nil {
+				return err
+			}
+			if genesisConfigFile != "" {
+				if args[1] == "" {
```

Comment:
Source URL is always necessary not just when providing config for initial genesis